node/malus/src/malus.rs

```diff
@@ -48,11 +48,24 @@ enum NemesisVariant {
 	PvfExecuteWorker(polkadot_cli::ValidationWorkerCommand),
 }
 
+#[derive(ArgEnum, Clone, Debug, PartialEq)]
+#[clap(rename_all = "kebab-case")]
+pub enum FakeCandidateValidation {
+	Invalid,
+	// TODO: impl Valid.
+}
+
 #[derive(Debug, Parser)]
 #[allow(missing_docs)]
 struct MalusCli {
 	#[clap(subcommand)]
 	pub variant: NemesisVariant,
+
+	#[clap(long, arg_enum, ignore_case = true)]
+	pub fake_backing_validation: Option<FakeCandidateValidation>,
+
+	#[clap(long, arg_enum, ignore_case = true)]
+	pub fake_approval_validation: Option<FakeCandidateValidation>,
```

Comment:
Shouldnttis be part of the variant?

Reply:
Hmm. I think you are right, I considered this initially but then I was thinking that this makes sense as a general configuration option. Now, I think if it's not implemented for all variants it will confuse users.